themes/cardinal_service/lib/components/mission-statement/mission_statement.scss

```diff
@@ -0,0 +1,54 @@
+@charset 'UTF-8';
+
+@import '../../scss/config';
+
+.su-mission-statement {
+  margin-left: calc(50% - 50vw);
+  margin-right: calc(50% - 50vw);
+
+  img,
+  picture,
+  source {
+    width: 100vw;
+  }
+
+  .background {
+    background: $su-color-brown;
+    position: relative;
+    min-height: 300px;
+  }
+
+  .overlay {
+    position: absolute;
+    top: 0;
+    color: #fff;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+  }
+
+  .overlay-text {
+    text-align: center;
+    margin: 0 auto;
+    max-width: 1426px;
```

Comment:
I'm curious about the max-width of 1426px

Reply:
Its just a placeholder for now. Rebecca will be styling it and make it look all better.